docs/dev.md

```diff
@@ -129,3 +129,4 @@ These parameters are available in addition to the ones in the `dev` section abov
 | container | If set to `true`, run the server in the container specified by the `dockerfile` parameter. Setting this to `true` is equivalent to using the `devc` goal. The default value is `false` when the `dev` goal is used, and `true` when the `devc` goal is used. | No |
 | dockerRunOpts | Specifies options to add to the `docker run` command when using dev mode to launch your server in a container. For example, `-e key=value` is recognized by `docker run` to define an environment variable with the name `key` and value `value`. Setting this parameter overrides the `container` parameter to `true`. | No |
 | dockerfile | Location of a Dockerfile to be used by dev mode to build the container that runs your server and to specify the context used to build the container image. The default value is `Dockerfile`. Setting this parameter overrides the `container` parameter to `true`. | No |
+| dockerBuildTimeout | Maximum time to wait (in seconds) for the completion of the docker operation to build the container. The value must be an integer greater than 0. The default value is `60` seconds. | No |
```

Comment:
Instead of "build the container", it should be "build the image" (or Docker image).

Reply:
You're right. I was so used to referring to the command docker but now it is a concept.